cmake/Modules/Findswig.cmake

```diff
@@ -0,0 +1,44 @@
+add_executable(swig IMPORTED)
```

Comment:
It would be great to set path to executable after configuration:
```Make
set_target_properties(swig PROPERTIES
    IMPORTED_LOCATION ${SWIG_EXECUTABLE}
    )
```

Reply:
O, that is a nice feature. Thanks!